amethyst_renderer/src/pipe/effect/mod.rs

```diff
@@ -226,7 +226,7 @@ impl<'a> EffectBuilder<'a> {
                 Stencil::default(),
             ));
         }
-        self.init.out_colors.push(name);
+        self.init.out_colors.push((name, ColorMask::all(), Blend::new(Equation::Add, Factor::One, Factor::OneMinus(BlendValue::SourceAlpha))));
```

Comment:
I'm not sure about this blend equation

Reply:
Aren't sum of `Factors` should be 1?

Reply:
You can use some of our existing [presets](https://docs.rs/draw_state/0.7.1/draw_state/preset/blend/constant.ALPHA.html).
Ideally, let the user (implementer of the graphics passes) manually set the blending settings. Similar to the depth settings above.

Reply:
It should be possible for custom passes. Passes for simple transparency should use specific blend function